awsshell/compat.py

```diff
@@ -10,11 +10,13 @@
 if PY3:
     from html.parser import HTMLParser
     text_type = str
+    long_type = int
     from io import StringIO
     import dbm
 else:
     from HTMLParser import HTMLParser
     text_type = unicode
+    long_type = long
```

Comment:
Would ``six`` help with some of these? Could help to notice that you have a ``text_type`` as well which six has. Furthermore, we really do not worry about longs in botocore so you can just get away with ``six.integer_types``: https://github.com/boto/botocore/blob/develop/botocore/validate.py#L235-L237